### DISCUSSION TITLE
<Link> element, Persistent Layouts and #each directive

### DISCUSSION THREAD

**Discussion by romek-codes** [inertiajs/inertia#1616] [Help]
Hi I've recently had an issue with using the <Link> element and persistent Layouts, when switching from a site that uses the #each directive.
You can read more about it in depth [here](https://stackoverflow.com/questions/76740112/uncaught-in-promise-error-each-only-iterates-over-array-like-objects-when?noredirect=1#comment135292699_76740112) on Stack. <p>I managed to solve it but i wanted to ask, is that the correct solution? It seems weird to attach values to responses that aren't really utilized by them, but I didn't find another way to fix the issue.</p>
<p>If anybody could tell me some other ways i could fix it I'd be thankful :)</p>

**Comment by ankushmak2** [comment]
yes you can do that
